rpc/manager/manager.proto

```diff
@@ -544,10 +544,25 @@ message CLIConfig {
   bytes config_yaml = 1;
 }
 
+enum Mode {
+  MODE_UNSPECIFIED = 0;
+  MODE_SINGLE = 1;
+  MODE_TEAM = 2;
+}
+
+message StatusInfo {
+  VersionInfo2 version = 1;
+  Mode mode = 2;
+  int32 client_count = 3;
+}
+
 service Manager {
   // Version returns the version information of the Manager.
   rpc Version(google.protobuf.Empty) returns (VersionInfo2);
 
+  // Status returns traffic manager state information.
```

Comment:
This is misleading as it returns a very small fraction of the manager's state.

Reply:
That begs the question, should it return more?